guides/paper-iron-layout.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Responsive layouts with Paper and Iron
+summary: "How to create responsive headers, tabs, side drawers, and 
+icons with Paper and Iron elements."
+tags: ['layout']
+elements: ['paper-header-panel','paper-toolbar','paper-drawer-panel',
+'paper-icon-button','paper-tabs','paper-tab','paper-drawer-panel', 'iron-icons',
+'iron-flex-layout']
+updated: 2015-07-23
+---
+
+## Introduction
+
+This guide teaches you how to use Paper and Iron elements to create a 
+responsive layout.
+
+## Installation
+
+Below is a list of commands for installing all of the elements mentioned
+in this document. You probably
+do not need to install all of these elements. Read the guide and decide
+how you want to implement your layout, and then install only the elements
+that you need.
+
+```bash
+bower install -S PolymerElements/paper-header-panel
```

Comment:
For consistency, we should probably use `--save` rather than `-S`, since that's the form we show in [getting the code](https://www.polymer-project.org/1.0/docs/start/getting-the-code.html) (and in the catalog GUI).